Styles/image.css

```diff
@@ -0,0 +1,23 @@
+.image-intro {
+    color: var(--paragraph-color);
+    letter-spacing: 1px;
+    line-height: 1.3;
+}
+
+.Image-note {
+    color: var(--primary-color-hover);
+}
+
+.responsive-image {
+    width: 100%;
+    height: auto;
+}
+
+.image-container {
+    border: 1px solid var(--border-color);
+    width: 100%;
+    height: 150px;
```

Comment:
you can use absolute vale rem for better responsive

Reply:
ok will change that